profiles/logical/hpos/development/default.nix

```
@@ -0,0 +1,5 @@
+{
+  imports = [ ../. ];
+
+  environment.variables.HPOS_DEVELOPMENT_MODE = true;
```

```suggestion
  environment.variables.HPOS_DEVELOPMENT_MODE = "true";
```
env vars can only be of a type `string`